src/__experimental__/components/radio-button/radio-button.stories.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import {
+  boolean, text, number, select
+} from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import OptionsHelper from '../../../utils/helpers/options-helper';
+import { RadioButton, RadioButtonGroup } from '.';
+import { info, notes } from './documentation';
+
+storiesOf('Experimental/RadioButton', module)
+  .add('default', () => {
+    const knobs = defaultKnobs();
+
+    return (
+      <RadioButtonGroup
+        groupName='frequency'
+        label={ text('groupLabel', 'Please select a frequency from the options below') }
+      >
+        <RadioButton
+          id='input-1'
+          label={ text('radioOneLabel', 'Example Weekly Radio Button') }
+          value={ text('radioOneValue', 'weekly') }
+          { ...knobs }
+        />
+        <RadioButton
+          id='input-2'
+          label={ text('radioTwoLabel', 'Example Monthly Radio Button') }
+          value={ text('radioTwoValue', 'monthly') }
+          { ...knobs }
+        />
+        <RadioButton
```

Comment:
Would `id='input-3'` be needed for this `<RadioButton>` ? (similar to how the first two `<RadioButton>`s have `id`s)

Reply:
Nope, the first two are intended to demonstrate the ability to provide manual values, for the third one it's intentionally left off, as this will prompt a GUID to be generated instead 😄

Reply:
Ah OK :thumbsup: Just a minor suggestion: Maybe it'd be helpful to add a comment in the code to explain this? It makes sense now that you've explained it, but it's not obvious just from looking at the code :smile:

Reply:
Thanks for adding that comment!